packages/plugins/plugin-assistant/src/capabilities/blueprint-definition.ts

```diff
@@ -47,7 +49,8 @@ const blueprint = createBlueprint();
 // TODO(dmaretskyi): Consider splitting into multiple modules.
 export default (): Capability<any>[] => [
   contributes(Capabilities.Functions, functions),
-  contributes(Capabilities.Functions, [research]),
+  contributes(Capabilities.Functions, [agent]),
+  contributes(Capabilities.Functions, [research, createResearchNote]),
```

Comment:
is there a reason not to put them in the same array?

Reply:
they are supposed to be grouped by the blueprint they come from. but im open to changing this